pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -568,13 +570,15 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         get(apiRoot + RoutePath.BSP_KIT_QUERY_PATH, new BSPKitRoute(notificationUtil), new JsonTransformer());
         get(apiRoot + RoutePath.BSP_KIT_REGISTERED, new BSPKitRegisteredRoute(), new JsonTransformer());
         get(apiRoot + RoutePath.CLINICAL_KIT_ENDPOINT, new ClinicalKitsRoute(notificationUtil), new JsonTransformer());
-        get(apiRoot + RoutePath.CREATE_CLINICAL_KIT_ENDPOINT, new CreateClinicalDummyKitRoute(new OncHistoryDetailDaoImpl()),
-                new JsonTransformer());
-        get(apiRoot + RoutePath.CREATE_CLINICAL_KIT_ENDPOINT_WITH_PARTICIPANT,
-                new CreateClinicalDummyKitRoute(new OncHistoryDetailDaoImpl()),
-                new JsonTransformer());
+
+        NotPepperKitCreationService notPepperKitCreationService = new NotPepperKitCreationService();
+        get(apiRoot + RoutePath.SHIP_KIT_ENDPOINT, new JuniperShipKitRoute(notPepperKitCreationService), new JsonTransformer());
 
         if (!cfg.getBoolean("ui.production")) {
+            get(apiRoot + RoutePath.CREATE_CLINICAL_KIT_ENDPOINT, new CreateClinicalDummyKitRoute(new OncHistoryDetailDaoImpl()),
+                    new JsonTransformer());
+            get(apiRoot + RoutePath.CREATE_CLINICAL_KIT_ENDPOINT_WITH_PARTICIPANT, new CreateClinicalDummyKitRoute(new OncHistoryDetailDaoImpl()),
+                    new JsonTransformer());
```

Comment:
Moving these 2 dummy routes out of production

Reply:
They are dummy kits created for GP tests last year, We normally take them out of production after the tests are completed.